src/classes/audioContextWrapper.tsx

```diff
@@ -10,12 +10,15 @@ class AudioContextWrapper {
     masterGainNode : GainNode
     filterNode : BiquadFilterNode
     waveshaperNode : WaveshaperNodeWrapper
-    oscilators: OscillatorWrapper[]
+    primaryOscillators: OscillatorWrapper[]
```

Comment:
Hmm - I disagree with this placement. Since the OscillatorWrapper itself is responsible for the spinning up and spinning down of the Oscillators, can this logic live there instead? So we'd still have our oscillators member here, and we'd pass the detune amount and the number of oscillators to spin up into the wrapper. Curious to hear what you think

Reply:
I ended up refactoring OscillatorWrapper into Voice.Let me know what you think.